CoseSignTool/SignCommand.cs

```diff
@@ -29,12 +29,28 @@ public class SignCommand : CoseCommand
         ["-sl"] = "StoreLocation",
         ["-ContentType"] = "ContentType",
         ["-cty"] = "ContentType",
+        ["-IntHeaders"] = "IntHeaders",
+        ["-ih"] = "IntHeaders",
+        ["-StringHeaders"] = "StringHeaders",

```

Comment:
Why not filter this down to just ProtectedHeaders and UnprotectedHeaders?
The code can tell if they are string or int and build the larger arrays, but the user presumably needs to decide if they are protected or unprotected.

Reply:
I did not do it as there is a possibility that users might want to represent an integer as a string in the header.